src/com/connexta/pom_lint.clj

```diff
@@ -41,22 +41,37 @@
        (map descriptor->depstruct)
        set))
 
-(defn main [fname]
-  (let [data (xml/parse fname)
-        root-deps (get-root-deps data)
-        all-deps (into (get-all-deps data) (get-all-descriptors data))]
+(defn get-all-features [data]
+  (->> (xml-seq data)
+       (filter #(= :bundle (:tag %)))
+       (map :content)
+       first
+       (map str/trim)
+       (map #(re-matches #"mvn:(.*)\/(.*)\/(.*)" %))
+       (map descriptor->depstruct)
```

Comment:
✏️ I think we should rename this fun since it's now being used for both pom descriptors and feature file bundles. Maybe, `mvncoor->depstruct`?

Reply:
Agreed! @djblue pointed this out as well so you must be on similiar wavelengths. 😄